docs/docs/cmd/spo/tenant/tenant-settings-set.mdx

```diff
@@ -14,257 +14,494 @@ m365 spo tenant settings set [options]
+: Enables or disables the link "Return to classic SharePoint" on modern SharePoint list and library pages. Allowed values `true`, `false`.
+
+`--DisablePersonalListCreation [DisablePersonalListCreation]`
+: Allows creating personal lists within the OneDrive for Business site of the user. Allowed values `true`, `false`.
+
+`--ViewInFileExplorerEnabled [ViewInFileExplorerEnabled]`
+: Enables or disables the ability to use View in Explorer in Microsoft Edge. Allowed values `true`, `false`.
+
+`--AllowGuestUserShareToUsersNotInSiteCollection [AllowGuestUserShareToUsersNotInSiteCollection]`
+: Allows guest users to share to users not in the site collection. Allowed values `true`, `false`.
+
+`--BlockSendLabelMismatchEmail [BlockSendLabelMismatchEmail]`
+: Allows blocking of the automated e-mail being sent when somebody uploads a document to a site that's protected with a sensitivity label and their document has a higher priority sensitivity label than the sensitivity label applied to the site. Allowed values `true`, `false`.
+
+`--CoreDefaultLinkToExistingAccess [CoreDefaultLinkToExistingAccess]`
+: Sets default share link to existing access on core partition. Allowed values `true`, `false`.
+
+`--CoreRequestFilesLinkEnabled [CoreRequestFilesLinkEnabled]`
+: Enables or disables the Request files link on the core partition for all SharePoint sites (not including OneDrive sites). If this value is not set, Request files link will only show for OneDrives with Anyone links enabled. Allowed values `true`, `false`.
+
+`--OneDriveRequestFilesLinkEnabled [OneDriveRequestFilesLinkEnabled]`
+: Enables or disables the Request files link on the OneDrive partition for all OneDrive sites. If this value is not set, Request files link will only show for OneDrives with Anyone links enabled. Allowed values `true`, `false`.
+
+`--DisableDocumentLibraryDefaultLabeling [DisableDocumentLibraryDefaultLabeling]`
+: Allows tenant admins to disable the capability of configuring a default sensitivity label for a document library. Allowed values `true`, `false`.
+
+`--DisableVivaConnectionsAnalytics [DisableVivaConnectionsAnalytics]`
+: Specifies whether Viva Connections Analytics is disabled or enabled on the tenant. Allowed values `true`, `false`.
+
+`--DisplayNamesOfFileViewersInSpo [DisplayNamesOfFileViewersInSpo]`
+: Allows configuring whether display name of people who view the file are visible in the property pane of the site in SharePoint site collection. Allowed values `true`, `false`.
+
+`--EnableAIPIntegration [EnableAIPIntegration]`
+:  Specifies whether Azure Information Protection (AIP) should be enabled on the tenant. Allowed values `true`, `false`.
+
+`--EnableRestrictedAccessControl [EnableRestrictedAccessControl]`
+: Enables restricted access control in SharePoint Online. You need to wait approximately 1 hour before managing restricted access control for a site collection. Allowed values `true`, `false`.
+
+`--HideSyncButtonOnDocLib [HideSyncButtonOnDocLib]`
```

Comment:
Can we place this option next to `HideSyncButtonOnODB` so they are grouped together?

Reply:
Pushed commit with required changes.